archive/100-theorems-list/16_abel_ruffini.lean

```diff
@@ -23,7 +23,7 @@ Then all that remains is the construction of a specific polynomial satisfying th
 
 -/
 
-namespace abel_ruffini
+namespace archive.abel_ruffini
```

Comment:
As discussed on Zulip, I don't think we want to put these in the `archive` namespace (yet). @digama0, can you confirm you agree the the inner namespace alone is sufficient and preferable?